packages/react/src/components/Table/mdx/RowExpansion.mdx

```diff
@@ -0,0 +1,91 @@
+View the full Table documentation [here](/docs/1-watson-iot-table--playground)
+
+## Row expansion
+
+The `Table` components support row-level expansion by setting `options.hasRowExpansion` to true. The content
+of the expanded rows must be provided ahead of time using the prop [expandedData](#expandeddata-prop).
```

Comment:
If you're looking at the row-expansion documentation this link doesn't work, because these props are only in the full table docs. Maybe a full link instead? `?path=/docs/1-watson-iot-table--playground#expandeddata-prop`

Reply:
Yes, good catch!